x-pack/plugins/ml/public/application/components/import_export_jobs/import_jobs_flyout/import_jobs_flyout.tsx

```diff
@@ -52,7 +52,7 @@ export const ImportJobsFlyout: FC<Props> = ({ isDisabled }) => {
   const {
     services: {
       data: {
-        indexPatterns: { getTitles: getIndexPatternTitles },
+        dataViews: { getTitles: getIndexPatternTitles },
```

Comment:
nit, might as well rename `getIndexPatternTitles` here too

Reply:
Done in https://github.com/elastic/kibana/pull/121517/commits/f23fcd0bf70913097b838e5ebb8a6f2560723639